engine/access/rpc/backend/backend_transactions.go

```diff
@@ -416,6 +418,8 @@ func (b *backendTransactions) GetTransactionResultsByBlockID(
 		"number of transaction results returned by execution node is less than the number of transactions  in the block",
 	)
 
+	eventEncodingVersion := convert.GetConversionEventEncodingVersion(eventEncodingVersionValue)
```

Comment:
same comment about verifying the from/to conversions for the tx result endpoints